pytest_dz/test_human.py

```diff
@@ -1,4 +1,68 @@
+import pytest
+
+
 def test_grow(create_human):
     human = create_human
     human.grow()
     assert human.age == 23
+
+
+def test_grow_reach_age_limit(create_giga_old_human):
+    human = create_giga_old_human
+    for _ in range(2):
+        human.grow()
+    assert human._Human__status == "dead", "Person should reach age limit and die"
+
+
+def test_grow_if_already_dead(create_giga_old_human):
+    human = create_giga_old_human
+    human._Human__status = "dead"
+    with pytest.raises(Exception, match="Misha is already dead..."):
+        human.grow()
+
+
+def test_is_alive_status_alive(create_human):
+    human = create_human
+    result = human._Human__is_alive()
+    assert result is True, "Person should be alive"
+
+
+def test_is_alive_status_dead(create_human):
```

Comment:
is_alive_status_dead what does it mean??

Reply:
> is_alive_status_dead what does it mean??
I wanted to test the statuses used in the private function so "_is_alive" function name and "status_dead" is status that i'd tested in this test, but it turns out that there is no point because this is covered by other tests which call the grow() method and consequently check this status